packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -170,9 +170,15 @@ export class SDKClient {
       ),
     );
 
-    // If the authorized fee from the Ethereum sender is insufficient, the payer of the transaction is charged up to the maxGasAllowance.
-    // see "Max Allowance" in the docs for more details https://docs.hedera.com/hedera/sdks-and-apis/sdks/smart-contracts/ethereum-transaction
-    ethereumTransaction.setMaxGasAllowanceHbar(ConfigService.get('MAX_GAS_ALLOWANCE_HBAR'));
+    const payMasterWhiteList = ConfigService.get('PAYMASTER_WHITELIST').map((e) => e.toLowerCase());
```

Comment:
We have this case in the acceptance criteria which I believe is not covered:

"When Paymaster flag is activated, whitelist needs to be present, if not node startup will giver error and halt start", perhaps it should be added. Maybe here or in `validationService.ts`?

Reply:
This might be related to https://github.com/hiero-ledger/hiero-json-rpc-relay/pull/3941#discussion_r2221351234, so I will put it on hold till a decision has been made.